core/src/main/java/hudson/model/Failure.java

```diff
@@ -54,6 +55,13 @@ public Failure(String message, boolean pre) {
         this.pre = pre;
     }
 
+    public void generateResponse(StaplerRequest req, StaplerResponse rsp, Object node, @Nullable Throwable throwable) throws IOException, ServletException {
```

Comment:
Better to use <code>CheckForNull</code>. Nullable is a weak definition, which does not require the code to really check for null.